specs/mgt-file-picker.md

```diff
@@ -1,57 +1,75 @@
-| Extended file picker is launched when “See all” is clicked | P2 | Not included in v1. |
+| Extended file picker is launched when “See all” is clicked | P0 | |
+| **v2** | | |
+| Retrieve files from a specified insight type or from a specific drive location based on identifiers provided by developer | P1 | |
+| Ability for developer to toggle between single-select and multiselect | P1 | Will not be included in v1. | 
+| Visual indication of selected files | P2 | |
+| Render the selected files | P2 | |
 
 ## Proposed Solution
 
-### Example 1: Show the user's top 10 most recently used files to select from
+### Example 1 (v1): Show the user's top 10 most recently used files to select from
 ```<mgt-file-picker insight-type=“used” show-max=“10”></mgt-file-picker>```
 
-### Example 2: Show a specific folder in OneDrive to select files from
+### Example 2 (v2): Show a specific folder in OneDrive to select files from
 ```<mgt-file-picker drive-id=“123” item-id=“456”></mgt-file-picker>```
 
 ## Attributes and Properties
 
 | Attribute | Property | Description |
 | --------- | -------- | ----------- |
-| `file-list-query` | `fileListQuery` | The full query or path to the folder in a drive or site to retrieve the list of files from |
-| `file-queries` | `fileQueries` | An array of file queries to be rendered by the component |
+| **v1** | | |
 | `files` | `files` | An array of files to get or set the list of files rendered by the component. Use this to access the files loaded by the component. Set this value to load your own files. |
 | `insight-type` | `insightType` | Set to show the user’s trending, used, or shared files. |
+| `show-max` | `showMax` | A number value to indicate the maximum number of files to show. |
+| **v2** | | |
+| `file-list-query` | `fileListQuery` | The full query or path to the folder in a drive or site to retrieve the list of files from |
+| `file-queries` | `fileQueries` | An array of file queries to be rendered by the component |
 | `drive-id` | `driveId` | Id of the drive the folder belongs to. Must also provide either `item-id` or `item-path`. |
 | `group-id ` | `groupId` | Id of the group the folder belongs to. Must also provide either `item-id` or `item-path`. |
 | `site-id` | `siteId` | Id of the site the folder belongs to. Must also provide either `{item-id}` or `{item-path}`. Provide `{list-id}` if you’re referencing a file from a list. |
 | `item-id` | `itemId` | Id of the folder. Default query is `/me/drive/items`. Provide `{drive-id}`, `{group-id}`, `{site-id}`, or `{user-id}` to query a specific location. |
 | `item-path` | `itemPath` | Item path of the folder. Default query is `/me/drive/root`. Provide `{drive-id}`, `{group-id}`, `{site-id}`, or `{user-id}` to query a specific location. |
-| `show-max` | `showMax` | A number value to indicate the maximum number of files to show. |
 | `selected-files` | `selectedFiles`| An array of selected file objects. |
```

Comment:
To me `selected-files` implies that the component maintains state between interactions. Unless it only has value while the picker is open and being used in multi-select mode. Is that the thought process, or should we perhaps remove it?